charts/rancher-csp-adapter/0.1.0+up0.1.0-rc2/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/kube-version: 0.1.0-rc2
+  catalog.cattle.io/namespace: cattle-csp-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/permits-os: linux
+  catalog.cattle.io/rancher-version: 0.1.0-rc2
```

Comment:
```suggestion
  catalog.cattle.io/rancher-version: 2.6.6
```

Reply:
Same as above on the kube version - something got messed up on either the publish or build script. We should definitely fix.